src/frontend/magnify/jest-setup.ts

```diff
@@ -1,3 +1,7 @@
 import '@testing-library/jest-dom';
 
 Object.defineProperty(window, 'scrollTo', { value: jest.fn, writable: true });
+
+module.exports = async () => {
+    process.env.TZ = 'UTC';
```

Comment:
Good idea. You can also define the environment variable `TZ` in the `test` script defined in your package.json

```
"test": "TZ=UTC jest"
```

Reply:
I cannot mock dates without that (or headaches). I think it is not only convenient, but required for some of my tests (I mock some dates, so at this time, I choose a TZ, so it must be specified everywhere). Do you think it is safe enough to put it in the scripts ? It can easily be forgotten when writing scripts for other environments, does it ?

Reply:
while you are using `yarn test` you will always use `TZ=UTC`. So no there is no risk but you can keep it like this.